auth/services/irma/irmaconfig.go

```diff
@@ -0,0 +1,123 @@
+package irma
+
+import (
+	"fmt"
+	"os"
+	"sync"
+
+	"github.com/nuts-foundation/nuts-node/auth/logging"
+
+	"github.com/pkg/errors"
+	irma "github.com/privacybydesign/irmago"
+	"github.com/privacybydesign/irmago/server"
+	"github.com/privacybydesign/irmago/server/irmaserver"
+)
+
+// IrmaMountPath contains location the irma webserver will mount
+const IrmaMountPath = "/auth/irmaclient"
```

Comment:
according to new urls this must be under `/public` >> create issue for irma paths migration

Reply:
https://github.com/nuts-foundation/nuts-node/issues/93